life_expectancy/tests/conftest.py

```diff
@@ -15,9 +18,25 @@ def run_before_and_after_tests() -> None:
     # Teardown : fill with any logic you want
     file_path = OUTPUT_DIR / "pt_life_expectancy.csv"
     file_path.unlink(missing_ok=True)
+    
+@pytest.fixture(scope="session")
+def eu_life_expectancy_raw() -> pd.DataFrame:
+
+    """Fixture to load the raw life expectancy data"""
+    dataframe = load_data("life_expectancy/data/eu_life_expectancy_raw.tsv")
```

Comment:
It is, and well done pointing that out! I have made the corrections, so that the fixtures are not dinamically created based on the functions I want to test